code/code/misc/tweaks.cc

```diff
@@ -0,0 +1,268 @@
+//////////////////////////////////////////////////////////////////////////
+//
+// SneezyMUD - All rights reserved, SneezyMUD Coding Team
+//
+//  tweaks.cc
+//
+//////////////////////////////////////////////////////////////////////////
```

Comment:
I think we're AGPL or somesuch, these days -- we reserve really few rights. Best remove this whole block